packages/marble-api/openapis/marblecore-api.yaml

```diff
@@ -5214,6 +5336,75 @@ components:
           type: integer
         rate_limit_duration:
           type: integer
+    TestRunStatusDto:
+      type: string
+      enum: ['pending', 'up', 'down', 'unknown']
+    TestRunDto:
+      type: object
+      required:
+        - id
+        - scenario_id
+        - ref_iteration_id
+        - phantom_iteration_id
+        - start_date
+        - end_date
+        - creator_id
+        - status
+      properties:
+        id:
+          type: string
+          format: uuid
+        scenario_id:
+          type: string
+          format: uuid
+        ref_iteration_id:
+          type: string
+          format: uuid
+        phantom_iteration_id:
+          type: string
+          format: uuid
+        start_date:
+          type: string
+          format: date-time
+        end_date:
+          type: string
+          format: date-time
+        creator_id:
+          type: string
+          format: uuid
+        status:
+          $ref: '#/components/schemas/TestRunStatusDto'
```

Comment:
This is the full list of what the frontend expect (according to the Figma).

Still, one thing is not included: an optionnal "log" info to informa user on the "why" an existing testrun transition to "down" status (ex: Manually cancelled, Cancelled due to live version update on the scenario, cancelled by another testrun....)

From the Figma, this part of the design (choice for now is to make this column out of scope for the first iteration on testrun):
![image](https://github.com/user-attachments/assets/8cfd3a55-f3e7-400a-9f0b-f0ebbae6dff6)

Reply:
works for me.